libs/expo/betterangels/src/lib/ui-components/RequestedProvidedServices/ServicesModal.tsx

```diff
@@ -195,46 +196,31 @@ export default function ServicesModal(props: IServicesModalProps) {
   };
 
   const closeModal = () => {
-    const newInitialServices = initialServices
-      // TODO: remove after cutover
-      .filter((item) => !!item.serviceEnum)
-      .filter((item) => item.serviceEnum !== ServiceEnum.Other)
-      .map((service) => ({ id: service.id, enum: service.serviceEnum! }));
-    const initialServiceOthers = initialServices
-      .filter((item) => item.serviceEnum === ServiceEnum.Other)
-      .map((service) => ({
-        id: service.id,
-        title: service.serviceOther || null,
+    const existing: SelectedService[] = initialServices
+      .filter((it) => it.service?.id)
+      .map((it) => ({
+        requestId: it.id,
+        serviceId: it.service!.id,
+        label: it.service?.label ?? '',
       }));
-    setServiceOthers(initialServiceOthers);
-    setServices(newInitialServices);
 
+    setServices(existing);
+    setServiceOthers([]);
     closeModalScreen();
   };
 
   useEffect(() => {
-    const newInitialServices = initialServices
-      // TODO: remove after cutover
-      .filter((item) => !!item.serviceEnum)
-      .filter((item) => item.serviceEnum !== ServiceEnum.Other)
-      .map((item) => ({
-        id: item.id,
-        enum: item.serviceEnum!,
-      }));
-    const initialServiceOthers = initialServices
-      .filter((item) => item.serviceEnum === ServiceEnum.Other)
-      .map((service) => ({
-        id: service.id,
-        title: service.serviceOther || '',
+    const existing: SelectedService[] = initialServices
```

Comment:
this needs to display other services already present on the interaction. 
list modal when creating: <img width="266" height="106" alt="image" src="https://github.com/user-attachments/assets/ec1da430-7bfc-4e67-96ef-b78c45094989" />
list modal when viewing again: <img width="257" height="69" alt="image" src="https://github.com/user-attachments/assets/a6fd3ee8-bcff-4022-a3a1-3f2fdb4b8cdd" />